tests/02_eradiate/01_unit/scenes/atmosphere/test_particle_layer.py

```diff
@@ -188,10 +188,10 @@ def test_particle_layer_eval_ckd(mode_ckd, tmpdir, test_particles_dataset, bins)
     )
     spectral_config = CKDMeasureSpectralConfig(bin_set="10nm", bins=bins)
     spectral_ctx = spectral_config.spectral_ctxs()[0]
-    assert np.isclose(layer.eval_sigma_t(spectral_ctx), 1.0 / ureg.km)
-    assert np.isclose(layer.eval_sigma_s(spectral_ctx), 0.8 / ureg.km)
-    assert np.isclose(layer.eval_sigma_a(spectral_ctx), 0.2 / ureg.km)
-    assert np.isclose(layer.eval_albedo(spectral_ctx).magnitude, 0.8)
+    assert np.allclose(layer.eval_sigma_t(spectral_ctx), 1.0 / ureg.km)
+    assert np.allclose(layer.eval_sigma_s(spectral_ctx), 0.8 / ureg.km)
+    assert np.allclose(layer.eval_sigma_a(spectral_ctx), 0.2 / ureg.km)
+    assert np.allclose(layer.eval_albedo(spectral_ctx).m, 0.8)
```

Comment:
Using `isclose()` here was intentional: the purpose was to prevent success if a vector is returned.